vstruct/primitives.py

```diff
@@ -724,3 +724,27 @@ def __repr__(self):
         base = "{%.8x-%.4x-%.4x-%.2x%.2x-%.2x%.2x%.2x%.2x%.2x%.2x}"
         return base  % self._guid_fields
 
+class enum_mixin(object):
```

Comment:
I wonder if we could keep it smaller by making enum=<CLASS> an optional arg to *all* int types ( obvs not enforcing value limits as that would mess up parsing "weird" things ) and display the enum if found or the int otherwise?